README.md

```diff
@@ -19,7 +19,11 @@ seems to be around 3x slower than implementations using Torch.
 
 `python neural_style.py --content <content file> --styles <style file> --output <output file>`
 
-(run `python neural_style.py --help` to see a list of all options)
+Run `python neural_style.py --help` to see a list of all options.
+
+Use `--checkpoint-output` and `--checkpoint-iterations` to save checkpoint images.
+
+Use `--iterations` to change the number of iterations (default 1000).  For a 528×960 pixel content file, 1000 iterations take around 30 minutes to run on a GPU, or 10 hours on a Mid 2013 Macbook Air.
```

Comment:
It would be better to:
* Use a more standard-sized content image (e.g. 512 x 512)
* Benchmark on a specific GPU and mention which GPU it was (there's huge variance between GPUs)
* Benchmark on a specific CPU and mention which CPU it was (rather than just the model of the computer, so people don't have to look it up)

I did a quick benchmark, and for 1000 iterations on a 512x512 image, it takes:
* 2.5 minutes on a GeForce GTX Titan X
* 1.5 hours on an Intel Core i7-5930K